great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -730,6 +902,105 @@ def _pandas_column_map_condition_values(
         return list(domain_values[: result_format["partial_unexpected_count"]])
 
 
+def _pandas_column_pair_map_condition_values(
+    cls,
+    execution_engine: PandasExecutionEngine,
+    metric_domain_kwargs: Dict,
+    metric_value_kwargs: Dict,
+    metrics: Dict[str, Any],
+    **kwargs,
+):
+    """Return values from the specified domain that match the map-style metric in the metrics dictionary."""
+    (
+        boolean_mapped_unexpected_values,
+        compute_domain_kwargs,
+        accessor_domain_kwargs,
+    ) = metrics["unexpected_condition"]
+    """
+    In order to invoke the "ignore_row_if" filtering logic, "execution_engine.get_domain_records()" must be supplied
+    with all of the available "domain_kwargs" keys.
+    """
+    domain_kwargs = dict(**compute_domain_kwargs, **accessor_domain_kwargs)
+    df = execution_engine.get_domain_records(
+        domain_kwargs=domain_kwargs,
+    )
+
+    if not ("column_A" in domain_kwargs and "column_B" in domain_kwargs):
+        raise ValueError(
+            """No "column_A" and "column_B" found in provided metric_domain_kwargs, but it is required for a column pair map metric
+(_pandas_column_pair_map_condition_values).
+"""
+        )
+
+    # noinspection PyPep8Naming
+    column_A_name = accessor_domain_kwargs["column_A"]
+    # noinspection PyPep8Naming
+    column_B_name = accessor_domain_kwargs["column_B"]
+
+    column_list = [column_A_name, column_B_name]
+
+    for column_name in column_list:
+        if column_name not in metrics["table.columns"]:
+            raise ge_exceptions.ExecutionEngineError(
+                message=f'Error: The column "{column_name}" in BatchData does not exist.'
+            )
+
+    domain_values = df[column_list]
+
+    domain_values = domain_values[boolean_mapped_unexpected_values == True]
+
+    result_format = metric_value_kwargs["result_format"]
+
+    if result_format["result_format"] == "COMPLETE":
+        return domain_values.to_dict("records")
+    else:
+        return domain_values[: result_format["partial_unexpected_count"]].to_dict(
+            "records"
+        )
```

Comment:
Shouldn't this be something other than records based on the contents of the standard test?

Reply:
@jcampbell You are correct.  This is fixed in a follow-on PR.  I will port the fix and re-commit.  This, in fact, resolves any issue with different output formats for the `unexpected_count` result.